jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -152,7 +152,7 @@ class DeploySamApplicationValidator {
 
     fun validateSettings(view: DeployServerlessApplicationPanel): ValidationInfo? {
         if (view.createStack.isSelected) {
-            validateStackName(view.newStackName.text)?.let {
+            validateStackName(view.newStackName.text, view.stacks.values())?.let {
```

Comment:
Should prolly just make `view` an instance variable of `DeploySamApplicationValidator` since they have the same lifecycle - then we don't need to pass the parameters around so much.

Reply:
Done.